ipatests/test_integration/test_acme.py

```diff
@@ -252,3 +298,55 @@ def test_disable_acme_service(self):
             ['curl', '--fail', self.acme_server],
             ok_returncode=22,
         )
+        result = self.master.run_command(['ipa-acme-manage', 'status'])
+        assert result.returncode == 0
```

Comment:
This check can be removed., the return code for status command will be 0 anyways.

Reply:
done